fuel-vm/src/interpreter/contract.rs

```diff
@@ -370,8 +372,8 @@ where
         .unwrap_or_default())
 }
 
-/// Increase the asset balance for a contract
-pub(crate) fn balance_increase<S>(
+/// Increase the asset balance for a contract.
+pub fn balance_increase<S>(
```

Comment:
I made it public because we can easily reuse it on the `fuel-core` side to update the contract balance.